Galaxus.Functional.NUnitExtension/Galaxus.Functional.NUnitExtension.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
On a more general note; we're using .nuspec files for the NuGet package building. Since we're gonna have two of those now, could you add one for the new project and move the existing .nuspec file to the main pproject's directory?